projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -389,16 +389,25 @@ public function override_svg_icons() {
 
 		$svg_items = array();
 		foreach ( $menu as $idx => $menu_item ) {
-			if ( count( $menu_item ) > 6 && 0 === strpos( $menu_item[6], 'data:image/svg+xml' ) && 'site-card' !== $menu_item[3] ) {
-				$svg_items[] = array(
-					'icon' => $menu[ $idx ][6],
-					'id'   => $menu[ $idx ][5],
+			if ( count( $menu_item ) < 7 ) {
```

Comment:
minor one, can you add a comment on why the `$menu_item` number needs to be greater than `6`?

Reply:
Yeah, good idea. It's obviously a legacy thing as it would be better to have those as named keys.

Reply:
thanks, Paul.